### PR TITLE
refactor(LoneScale List Node)!: Rename to LoneScale

### PR DESCRIPTION
**Pull request by Joffcom**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

**Review by agobrech — DISMISSED** [review]
LGTM

**Comment by Joffcom** [comment]
Waiting for approval from Polina before merging

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@0.232.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.232.0)

**Pull request merged**
